sat/session.py

```diff
@@ -25,145 +25,42 @@
 OAuth2 authentication support.
 """
 
-import json
 import logging
-import os.path
-import os
 
-from oauthlib.oauth2 import (UnauthorizedClientError, MissingTokenError,
-                             InvalidGrantError, LegacyApplicationClient)
-from requests_oauthlib import OAuth2Session
+from csm_api_client.session import UserSession
 
-from sat.cached_property import cached_property
 from sat.config import get_config_value
 from sat.util import get_resource_filename
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-class SATSession:
-    """Manage API sessions, authentication, and token storage/retrieval."""
-
-    TOKEN_URI = '/keycloak/realms/{}/protocol/openid-connect/token'
-    tenant = 'shasta'
-    client_id = 'shasta'
+class SATSession(UserSession):
```

Comment:
Why are the changes in this file showing up here in addition to in #59? It seems like they belong just in #59, unless I'm missing something.

Reply:
I am unsure -- I will try rebasing against the new `CRAYSAT-1569-use-api-client-library` branch content to see if that fixes things.